clearwater-etcd/usr/share/clearwater/bin/poll_etcd.sh

```diff
@@ -49,7 +49,15 @@ if [ -n "$1" ] && [ $1 == "--quorum" ]; then
   output="\"key\":\"$key\",\"value\":\"True\""
 else
   path="http://${management_local_ip:-$local_ip}:4000/v2/stats/self"
-  output="\"name\":\"${management_local_ip:-$local_ip}\""
+  if [ ! -z $etcd_cluster ]; then
```

Comment:
It's a good idea to quote variable expansions when using `[  ]` just in case the variable is blank (which can lead to unexpected bugs).